components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/util/NotificationUtil.java

```diff
@@ -556,40 +565,83 @@ public static Notification buildNotification(Event event, Map<String, String> pl
+    private static void resolveOrgNameAndOrgId(String tenantDomain, Map<String, String> placeHolderData)
+            throws IdentityEventException {
 
-        String organizationName = tenantDomain;
+        placeHolderData.put(ORGANIZATION_ID_PLACEHOLDER, null);
+        placeHolderData.put(ORGANIZATION_NAME_PLACEHOLDER, tenantDomain);
         try {
             if (MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
-                return organizationName;
+                return;
             }
             RealmService realmService = NotificationHandlerDataHolder.getInstance().getRealmService();
             int tenantId = IdentityTenantUtil.getTenantId(tenantDomain);
             Tenant tenant = realmService.getTenantManager().getTenant(tenantId);
-            if (tenant == null) {
-                return organizationName;
-            }
-            String associatedOrganizationUUID = tenant.getAssociatedOrganizationUUID();
-            if (StringUtils.isBlank(associatedOrganizationUUID)) {
-                return organizationName;
+            if (tenant == null || StringUtils.isBlank(tenant.getAssociatedOrganizationUUID())) {
+                return;
             }
-            OrganizationManager organizationManager =
-                    NotificationHandlerDataHolder.getInstance().getOrganizationManager();
-            organizationName = organizationManager.getOrganizationNameById(associatedOrganizationUUID);
+
+            placeHolderData.put(ORGANIZATION_NAME_PLACEHOLDER,
+                    getOrganizationManager().getOrganizationNameById(tenant.getAssociatedOrganizationUUID()));
+            placeHolderData.put(ORGANIZATION_ID_PLACEHOLDER, tenant.getAssociatedOrganizationUUID());
         } catch (OrganizationManagementClientException e) {
             if (!ERROR_CODE_ORGANIZATION_NOT_FOUND_FOR_TENANT.getCode().equals(e.getErrorCode())) {
                 throw new IdentityEventException(e.getMessage(), e);
             }
         } catch (OrganizationManagementException | UserStoreException e) {
             throw new IdentityEventException(e.getMessage(), e);
         }
-        return organizationName;
+    }
+
+    /**
+     * Fetch the branding preferences from the organization hierarchy by traversing from the base org to the super org.
+     * If branding preferences enabled organization is found, that will be used as the branding preference.
+     *
+     * @param ancestorOrgIds The list of ancestor org ids including the base organization itself.
+     * @param brandingPreferenceManager The branding preference manager.
+     * @return Branding preferences of the immediate organization which has configured branding preferences.
+     */
+    private static BrandingPreference fetchBrandingPreferenceFromOrgHierarchy(List<String> ancestorOrgIds, BrandingPreferenceManager brandingPreferenceManager) {
```

Comment:
Check on the character length

Reply:
Yes. This repo doesn't enable check style. Hence lot of check style issues found. Will try to do proper styling for the new code which is going to introduced .